includes/classes/SupportMonitor/Monitor.php

```diff
@@ -472,6 +472,17 @@ public function send_daily_report() {
 			),
 		];
 
+		// If this is production, request that a web vitals insight also be generated.
+		if ( 'yes' === $setting['production_environment'] ) {
+			$messages[] = $this->format_message(
+				[
+					'url' => 'https://10up.com/',
```

Comment:
@christianc1 Is this hard-coded 10up URL intentional?

Reply:
Thanks for catching that, I had it hardcoded because my testing environment isn't public to the internet and needed an URL that was for testing.  Updated in d7d69f0.